src/transport.js

```diff
@@ -67,7 +68,7 @@ module.exports = function (swarm) {
               log('dial canceled: %s', multiaddr.toString())
               // clean up already done dials
               if (conn) {
-                conn.close()
+                pull(pull.empty(), conn)
```

Comment:
This is just a quick patch for tests, it won't close connections as the otherside will be waiting for the multistream handshake to finish.